### DISCUSSION TITLE
Multiconnections

### DISCUSSION THREAD

**Discussion by nobilik** [bcakmakoglu/vue-flow#60] [Q&A] [answered]
Hi! Is it possible to connect multiple edges to one target handle? In my case it will be very useful.

**Comment by bcakmakoglu** [comment, marked as answer]
You can create multiple connections to the same node.
If your node has multiple source/target handles, you need to assign them an id and use that to create a connection/edge:
```ts
const edge =  {
    id: 'e1-2',
    targetHandle: '2a',
    sourceHandle: '1a',
    source: '1',
    target: '2',
  }
```

```vue | pure
<template #node-custom1>
  <Handle id="1a" type="source" :position="Position.Right" />
	Node 1
  <Handle type="target" :position="Position.Top" />
</template>
<template #node-custom2>
  <Handle type="source" :position="Position.Right" />
	Node 2
  <Handle id="2a" type="target" :position="Position.Top" />
</template>
```

**Reply by nobilik** [reply]
This is not what I mean. I understand that a node can have many handles, but in my case I need to connect the edge to handle, that already connected to somewhere. I'll explain...  I have one node that have many condition blocks inside and each of this blocks have own source handle. So, what if I want to connect N handles to one node and M handles to another? I don't know values of N and M and it's not very nice to create multiple handles on target node. 

UPD Ups... sorry) it works exactly like that ))) it's Sunday,  I should stop working))))

**Reply by bcakmakoglu** [reply]
Here's a [sandbox](https://codesandbox.io/s/youthful-rain-zhypko?file=/src/components/Basic.vue) demonstrating the above example, if you wanna play around with it :)
